content/developers/developer-patterns/navigating-merklelogs/index.md

```diff
@@ -537,10 +442,7 @@ The peak stack is [6, 9]
 
 ### massif 4
 
-Note that this case is particularly interesting because it completes a full
-cycle from one perfect power sized tree to the next. It is a fact of the MMR
-construction that the look back is never further than the most recent 'perfect'
-tree completing massif.
+Note that this case is particularly interesting because it completes a full cycle from one perfect power sized tree to the next. It is a fact of the MMR construction that the look back is never further than the most recent 'perfect' tree completing massif.
```

Comment:
```suggestion
Note that this case is particularly interesting because it completes a full cycle from one perfect power-sized tree to the next.
A fact of the MMR construction is the look back is never further than the most recent 'perfect' tree completing massif.
```